.gitignore

```diff
@@ -33,8 +34,23 @@ tests/resources/dbs/.db/**
 **/app_modules/**
 workbench/app_modules
 
+# webpack working compilation assets
+includes/js
+includes/rev-manifest.json
```

Comment:
Why not just add `**/rev-manifest.json` to cover all instances?

Reply:
Done!